python/src/pywebvue/core.py

```diff
@@ -350,7 +350,7 @@ def protocol_call(self, method, *args, **kwargs):
     # Start server
     # -------------------------------------------------------------------------
 
-    def run_server(self, port=None, thread=False):
+    def run_server(self, port=None, thread=False, exec_mode="main"):
```

Comment:
can you add **kwargs here too and forward it to wslink?

Reply:
I would even remote `exec_mode` and just let `**kwargs` glob all of the extra ones.

Reply:
Yes I can!